examples/record_loader.rb

```diff
@@ -1,18 +1,31 @@
 class RecordLoader < GraphQL::Batch::Loader
-  def initialize(model, column: model.primary_key, where: nil)
+  EMPTY_ARRAY = [].freeze
+
+  def initialize(model, column: model.primary_key, where: nil, unique: true)
     @model = model
-    @column = column.to_s
-    @column_type = model.type_for_attribute(@column)
+    @column = column
+    @column_type = model.type_for_attribute(@column.to_s)
```

Comment:
* a `sequel model` has no `type_for_attribute` method

Reply:
Right, that is why in https://github.com/Shopify/graphql-batch/issues/82#issuecomment-388565399 I said

> Likely the column type casting would still need to be removed to work with Sequel.

Reply:
What would that look like with Sequel?